conf.yaml

```diff
@@ -1417,10 +1417,22 @@ contents:
                 repo:   enterprise-search-pubs
                 path:   workplace-search-docs
           -
-
     -
-        title:      SIEM: Security Operations with the Elastic Stack
+        title:      Elastic Security
         sections:
+          -
+            title:      Elastic Endpoint Security
+            prefix:     en/endpoint/guide
+            current:    7.6
+            branches:   [ master, 7.x, 7.6 ]
```

Comment:
```
13:08:04 INFO:build_docs:fatal: pathspec 'docs/en/endpoint' did not match any files
```
This is failing because you're attempting to build the `7.x` and `7.6` branch without anything existing there. https://github.com/elastic/stack-docs/pull/895 Needs to be backported before this can succeed.

Reply:
Done.